randovania/games/fusion/logic_database/Main Deck.txt

```diff
@@ -1344,8 +1344,13 @@ Extra - room_id: [51]
   * Extra - door_idx: (116,)
   > Beside Pickup
       Any of the following:
-          Combat (Advanced) or Can Kill Tough Beam-Resistant Enemy
+          Screw Attack or Combat (Advanced)
```

Comment:
Screw attack is already accounted for in the template for Beam-Resistant Enemies, any reason you are adding it again or was just missed?

Reply:
I added can kill just underneath this, because SA and PBs are the only options to do so safely and easily. Thus,
Combat (Advanced) or
SA or
Can Kill and [expect dmg or Combat (Int)] or
4PBs and beginner trick.